lib/auth/auth.go

```diff
@@ -111,6 +111,8 @@ import (
 	"github.com/gravitational/teleport/lib/gcp"
 	"github.com/gravitational/teleport/lib/githubactions"
 	"github.com/gravitational/teleport/lib/gitlab"
+	"github.com/gravitational/teleport/lib/integrations/awsra"
+	"github.com/gravitational/teleport/lib/integrations/awsra/createsession"
```

Comment:
Does `lib/auth` already depend on AWS SDK? I suspect it does, but always worth asking since it's a huge package and making it depend on another huge dependency might impact binary size, linking times, etc.